src/library/scala/collection/SortedOps.scala

```diff
@@ -38,13 +38,33 @@ trait SortedOps[A, +C] {
     *
     *  @param from The lower-bound (inclusive) of the ranged projection.
     */
-  def from(from: A): C = rangeImpl(Some(from), None)
+  @deprecated("Use rangeFrom", "2.13.0")
+  final def from(from: A): C = rangeFrom(from)
+
+  /** Creates a ranged projection of this collection with no upper-bound.
+   *
+   *  @param from The lower-bound (inclusive) of the ranged projection.
+   */
+  def rangeFrom(from: A): C = rangeImpl(Some(from), None)
 
   /** Creates a ranged projection of this collection with no lower-bound.
     *
     *  @param until The upper-bound (exclusive) of the ranged projection.
     */
-  def until(until: A): C = rangeImpl(None, Some(until))
+  @deprecated("Use rangeUntil", "2.13.0")
+  final def until(until: A): C = rangeUntil(until)
+
+  /** Creates a ranged projection of this collection with no lower-bound.
+   *
+   *  @param until The upper-bound (exclusive) of the ranged projection.
+   */
+  def rangeUntil(until: A): C = rangeImpl(None, Some(until))
+
+  /** Create a range projection of this collection with no lower-bound.
+    *  @param to The upper-bound (inclusive) of the ranged projection.
+    */
+  @deprecated("Use rangeTo", "2.13.0")
+  final def to(to: A): C = rangeTo(to)
```

Comment:
Do we have a simple test like `List(1, 2, 3).to(ImmutableArray)` to check that the presence of the overload does not make the implicit conversion fail? (the implicit conversion is required to convert `ImmutableArray` value to a value of type `Factory[Int, ImmutableArray[Int]]`)

Reply:
The new overload is only present in subtypes of `SortedOps`, so `List` is not affected.

I didn't see any issues with the overload

```
scala> import scala.collection.immutable._
import scala.collection.immutable._

scala> SortedSet(1, 2, 3).to(BitSet)
res7: scala.collection.immutable.BitSet = BitSet(1, 2, 3)

scala> SortedSet(1, 2, 3).to(Immutable)
Immutable   ImmutableArray

scala> SortedSet(1, 2, 3).to(ImmutableArray)
res9: scala.collection.immutable.ImmutableArray[Int] = ImmutableArray(1, 2, 3)

scala> SortedSet(1, 2, 3).to(collection.mutable.WrappedArray)
res10: scala.collection.mutable.WrappedArray[Int] = WrappedArray(1, 2, 3)

scala> SortedSet(1, 2, 3).to(1)
                          ^
       warning: method to in trait SortedOps is deprecated (since 2.13.0): Use rangeTo
res11: scala.collection.immutable.SortedSet[Int] = TreeSet(1)



scala> import Ordering.Implicits._
import Ordering.Implicits._

scala> SortedSet(List(1,2,3))
res17: scala.collection.immutable.SortedSet[List[Int]] = TreeSet(List(1, 2, 3))

scala> SortedSet(List(1,2,3)).to(ImmutableArray)
res18: scala.collection.immutable.ImmutableArray[List[Int]] = ImmutableArray(List(1, 2, 3))

scala> SortedSet(List(1,2,3)).to(List)
res19: List[List[Int]] = List(List(1, 2, 3))

scala> SortedSet(List(1,2,3)).to(List())
                              ^
       warning: method to in trait SortedOps is deprecated (since 2.13.0): Use rangeTo
res20: scala.collection.immutable.SortedSet[List[Int]] = TreeSet()
```

Reply:
So, actually both `to` can co-exist at the same time. I don’t remember why I had to rename the one in `SortedOps` to `rangeTo`, maybe that was because our code was cross-compiled by older Scala version which were not working with the overloads.

So, in the end, we could even keep `to`, `from` and `until` as they were (no need to rename them to `rangeTo`, `rangeFrom` and `rangeUntil`), right?

Reply:
Renaming is an improvement anyway, in my opinion. And having an overloaded `to` that does a completetly different thing is very weird. So I'd go ahead with this change.

Reply:
OK!